onnxruntime/python/tools/transformers/models/sam2/convert_to_onnx.py

```diff
@@ -105,41 +125,36 @@ def parse_arguments():
     return args
 
 
-def main():
-    args = parse_arguments()
+def optimize_sam2_model(onnx_model_path, optimized_model_path, use_gpu: bool, float16: bool):
+    print(f"Optimizing {onnx_model_path} to {optimized_model_path} with float16={float16} and use_gpu={use_gpu}...")
 
-    checkpoints_dir = os.path.join(args.sam2_dir, "checkpoints")
-    sam2_config_dir = os.path.join(args.sam2_dir, "sam2_configs")
-    if not os.path.exists(args.sam2_dir):
-        raise FileNotFoundError(f"{args.sam2_dir} does not exist. Please specify --sam2_dir correctly.")
+    # Import from source directory.
+    transformers_dir = os.path.normpath(os.path.join(os.path.dirname(__file__), "..", ".."))
+    if transformers_dir not in sys.path:
+        sys.path.insert(0, transformers_dir)
```

Comment:
Can we recommend users to run the SAM2 scripts with `python -m models.sam2.convert_to_onnx <...>` similar to how the other folder scripts are called now? Then we can remove this import.

Reply:
I would like to keep it as it is, so that user can run it in current directory directly.